src/transforms.jl

```diff
@@ -94,4 +94,5 @@ reapply(transform::StatelessGeometricTransform, object, cache) =
 # ----------------
 
 include("transforms/stdcoords.jl")
-include("transforms/smoothing.jl")
\ No newline at end of file
+include("transforms/smoothing.jl")
+include("rotations.jl")
```

Comment:
These changes are not related to the Torus addition. Please try to start a new Git branch before submitting a PR.